lib/vtls/openssl.c

```diff
@@ -1882,6 +1882,8 @@ static void ossl_close(struct Curl_cfilter *cf, struct Curl_easy *data)
       int nread, err;
       long sslerr;
 
+      /* Do a read to catch any pending data in SSL or on the socket */
```

Comment:
I think that the original comment was better - if something is read, it's probably a close notify alert, not just "data".

Reply:
Thanks. Resurrected the comment out of curl-7_86_0.